spec/shortcuts_spec.rb

```diff
@@ -142,6 +142,14 @@
       m2 = NMatrix.random(2,:seed => 62)
       m3 = NMatrix.random(2,:seed => 65)
 
+      expect(m1.stype).to eq(:dense)
+      expect(m1.dtype).to eq(:float64)
+      
+      expect(m2.stype).to eq(:dense)
```

Comment:
Okay, I remember commenting and asking you why you took these lines out (can't find that now, hopefully not my imagination). I was actually hoping you'd answer instead of just putting them back. But I'll answer my own question instead:

It seems to me that they're included in an earlier test which simply verifies that `#random` produces the correct dtype and stype for output.

Once that test has been run, though, it's redundant for you to test that same thing on m1, m2, and on m3. Even if that other test were deleted (maybe wise to delete it, too), you only need to run the test on *one* of m1, m2, and m3; if it's true for one, it's almost certainly going to be true for the others.